laravel/resources/views/instances.blade.php

```diff
@@ -47,7 +47,7 @@
     <div class="row">
         <div class="col-lg-12">
             <div class="alert alert-primary" role="alert">
-                There is no instance configured yet! <a href="#modalAddInstance" data-bs-toggle="modal" data-bs-target="#modalAddInstance">Add a new instance now.</a>
+                There is no instance configured yet! <button class="btn btn-link p-0" data-bs-toggle="modal" data-bs-target="#modalAddInstance">Add a new instance now.</button>
```

Comment:
[CONCERN]

The `@can()` directive is missing here.

Reply:
This is the same like banner.blade

```
<div class="container">
    @can('add instances')
        <div class="row">
            <div class="col-lg-3">
                <button type="button" class="btn btn-primary btn-sm fw-bold" data-bs-toggle="modal" data-bs-target="#modalAddInstance">
                    Add Instance
                </button>
            </div>
        </div>
        <hr>
    @endcan
    @include('inc.standard-alerts')
</div>
´´´